shared/oae/api/oae.api.util.js

```diff
@@ -1866,7 +1866,5 @@ define(['exports', 'require', 'jquery', 'underscore', 'oae.api.config', 'markdow
      *
      * @return {String}   Event(s) that can fire when the content of an input field changes
      */
-    var getFormChangeEventNames = exports.getFormChangeEventNames = function() {
-        return $('html').hasClass('ie-lt10') ? 'change keyup paste cut' : 'change input';
-    };
+    var getFormChangeEventNames = exports.getFormChangeEventNames = return "change input";
```

Comment:
@sallakarppinen 
> What output did you get from the `git push origin HEAD --force` command?

Here's the output after executing that command:
```
Counting objects: 36, done.
Delta compression using up to 8 threads.
Compressing objects: 100% (27/27), done.
Writing objects: 100% (36/36), 2.81 KiB | 0 bytes/s, done.
Total 36 (delta 24), reused 0 (delta 0)
remote: Resolving deltas: 100% (24/24), completed with 24 local objects.
To https://github.com/fortMaximus/3akai-ux
 23a0d2d..68317d6  HEAD -> master
```
>  If your local master branch is correct now, try `git push origin master --force`?

Can you explain why I should do this?